tests/stm32_eth_lwip/README.md

```diff
@@ -0,0 +1,21 @@
+Tests for stm32 Ethernet periph driver
```

Comment:
same here, can we reuse another lwip test application, or make this a generic "lwip_udp_echo" application that pulls in stm32_eth using ```FEATURES_OPTIONAL += stm32_eth```?